stl/inc/xstring

```diff
@@ -4685,39 +4964,59 @@ basic_ostream<_Elem, _Traits>& operator<<(
 // basic_string LITERALS
 inline namespace literals {
     inline namespace string_literals {
-        _NODISCARD inline string operator"" s(const char* _Str, size_t _Len) {
+        _NODISCARD
+#ifndef __EDG__ // TRANSITION, VSO-1273381
+        _CONSTEXPR20_CONTAINER
+#endif // __EDG__

```

Comment:
We are loosing `inline` here for EDG, is that relevant?

Reply:
I don't think it's a huge deal, but I'll add an `#else inline` for the sake of consistency 😸 Impressive eagle eyes 👀